crypto/rand/rand_lcl.h

```diff
@@ -128,7 +128,12 @@ extern RAND_BYTES_BUFFER rand_bytes;
 extern RAND_DRBG rand_drbg;
 extern RAND_DRBG priv_drbg;
 
-/* How often we've forked (only incremented in child). */
+/*
+ * A "generation count" of forks.  Increment-only, and only ever written to
+ * in the child process of the fork, which is guaranteed to be single-threaded.
+ * As such, no locking is needed for normal (read) accesses, since the rest of
+ * fork processing is assumed to introduce the necessary memory barriers.
```

Comment:
Yes it is not significant.  As long as the "number I've times I've been a child" doesn't match "number of times the process has created a child" then we reseed.